src/python/WMCore/MicroService/Unified/MSOutput.py

```diff
@@ -44,24 +59,70 @@ def __init__(self, msConfig, logger=None):
         self.msConfig.setdefault("enableDataPlacement", False)
         self.msConfig.setdefault("excludeDataTier", ['NANOAOD'])
         self.msConfig.setdefault("rucioAccount", 'wma_test')
+        self.msConfig.setdefault("mongoDBUrl", 'mongodb://localhost')
+        self.msConfig.setdefault("mongoDBPort", 8230)
+        self.msConfig.setdefault("streamerBufferFile", "/tmp/msOutput/requestRecords")
```

Comment:
Yes, it is for the moment how I feed the streamer. I would pretty much like to keep the parameter there though, but will put the default value to `None`. So that once the streamer fails to open the file it would immediately fall back to reading the requests from memory buffer (which have been passed to the constructor too [2], but for my tests was ignored because I provided the file artificially at the path in question).
[1]
https://github.com/dmwm/WMCore/blob/3a8ee05954b2c217f3d7080faf331d76ee82452e/src/python/WMCore/MicroService/Unified/MSOutputStreamer.py#L54-L62

[2]
https://github.com/dmwm/WMCore/blob/3a8ee05954b2c217f3d7080faf331d76ee82452e/src/python/WMCore/MicroService/Unified/MSOutput.py#L172

Reply:
Upon a second thought I can see your point. Having it configured here to point the proper place is good. But I am trying to avoid confusion until we fully develop this part. Because if the file we point to for some reason shows up into the system but is not the expected one it will substitute what ever is given in the memory buffer. I will keep it with `None` for the moment, but yes the correct path should be exactly:

`/data/srv/state/reqmgr2ms/ms-output/requestRecords` or `/data/srv/state/reqmgr2ms/ms-output/streamerBufferFile`